src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ProjectCard.kt

```diff
@@ -0,0 +1,18 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import javafx.geometry.Pos
+import org.wycliffeassociates.otter.common.data.model.Project
+import tornadofx.*
+
+class ProjectCard(project : Project, openProject: ()-> Unit) : Fragment(){
```

Comment:
Based on what I've read about creating JavaFX UI controls, you'd want to extend a layout here, not a Fragment, even though I suppose this should also work?

ultimately though, this is what should implement project card (it is the project card after all)